integration/kubernetes/init.sh

```diff
@@ -67,6 +67,17 @@ esac
 # Check no there are no kata processes from previous tests.
 check_processes
 
+# Verify that there are not previous kubernetes installations
+kubernetes_file_path="/etc/kubernetes"
+if [ -d "${kubernetes_file_path}/manifests" ]; then
+	sudo rmdir "${kubernetes_file_path}/manifests"
```

Comment:
Are we sure the director will be empty?  Wouldn't be better to simply do a `rm -rf`? :-)